src/lib/stylesheet.js

```diff
@@ -216,7 +216,7 @@ export default class StyleSheet {
     return this._tags.reduce((rules, tag) => {
       if (tag) {
         rules = rules.concat(
-          this.getSheetForTag(tag).cssRules.map(rule =>
+          Array.from(this.getSheetForTag(tag).cssRules).map(rule =>
```

Comment:
Maybe go for the `Array.prototype.map.call(this.getSheetForTag(tag).cssRules, rule => {})` route so that when transpiling babel doesn't inline the polyfill for Array.from

Reply:
Of course. Patch upcoming.